wire-ios-request-strategy/Sources/E2EIdentity/ACME/AcmeAPITests.swift

```diff
@@ -272,16 +272,27 @@ class AcmeAPITests: ZMTBaseTest {
             httpVersion: "",
             headerFields: nil
         )!
-        let mockData = Data()
+
+        let mockCertificates = [
+            "certificate_1",
+            "certificate_2",
+            "certificate_3"
+        ]
+        let mockPayload = [
+            "crts": mockCertificates
+        ]
+
+        let mockData = try JSONSerialization.data(withJSONObject: mockPayload)
```

Comment:
Do we need to use serialization here? Midterm I would like to replace it with Data and Decodable.
In other tests usually define a json string and use .utf8 to make Data object.

Reply:
It was simply faster to write it this way rather than using a json string. I could also create a `FederationCertificates` object and encode it to data. What do you think?
Also, what's the reason against using serialisation?

Reply:
Updated https://github.com/wireapp/wire-ios/pull/1120/commits/c3d1d3a6dcbb02ee4c98be0a37b8d237862ffa66

Reply:
Thanks!